examples/marko-cli/package.json

```diff
@@ -19,8 +19,7 @@
   },
   "dependencies": {
     "marko": "^4.12.4",
-    "marko-starter": "^2.0.4",
-    "marko-widgets": "^7.0.1"
```

Comment:
why delete the widgets?

Reply:
They are deprecated and don't support marko 4

Reply:
okay, clear!